Lesson2/src/Homework.java

```diff
@@ -0,0 +1,89 @@
+import java.util.Scanner;
+
+public class Homework {
+
+    public static void main(String[] args) {
+        System.out.println(sum(10, 20));
+        System.out.println(sum(Integer.MAX_VALUE, Integer.MAX_VALUE));
+        System.out.println(max(50, 49));
+        System.out.println(average(new int[]{1, 2, 3, 4, 5}));
+        System.out.println(max(new int[]{1, 2, 3, 4, 5, 100, 99}));
+        System.out.println(calculateHypotenuse());
+    }
+
+    /**
+     * 1. Method should return sum of "a" and "b"
+     * 2. (optional) if sum of a and b is more then int can store -- return -1;
+     **/
+    public static int sum(int a, int b) {
+        return (a + b < 0 ? -1 : a + b);
```

Comment:
if a = -10, b = -20 then expected result will be -30, but this method will return -1
Good use of ternary operator though)

Reply:
Somebody in the chat wrote that condition is doesn't change, and we won't use -10 and -20.)) I thought that I need to use only integer type.

Reply:
int range is from -2147483648 to 2147483647, so that's true, we use only integer type